src/Compiler/CompilerDefines.h

```diff
@@ -105,6 +105,7 @@ static constexpr const char *GLOBAL_PLACEMENT_LOG{"global_placement.rpt"};
 static constexpr const char *PLACEMENT_LOG{"placement.rpt"};
 static constexpr const char *PLACEMENT_TIMING_LOG{"post_place_timing.rpt"};
 static constexpr const char *ROUTING_LOG{"routing.rpt"};
+static constexpr const char *ROUTING_TIMING_LOG{"post_rout_timing.rpt"};
```

Comment:
Typo: e missing in "post_rout**e**_timing.rpt"

Reply:
nice catch. fixed